### PR TITLE
Send messages through kafka proxy

### PR DESCRIPTION
**Pull request by BerniVarga**
The bridge is able to write messages to kafka through:
* kafka-proxy (using a message-producer-go-library)
* a plain http producer (used in the case of the cms-notifier)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-###Kafka consumer listening to kafka-proxy and forwarding messages to an HTTP endpoint.
+###Kafka consumer listening to kafka-proxy and forwarding messages to another KAFKA-PROXY or a simple HTTP endpoint.
 
 Run: `./start.sh`
 
-* Change parameters in start.sh, or set the environment variables ($QUEUE_PROXY_ADDRS, $GROUP_ID, $AUTHORIZATION_KEY, $HTTP_HOST) for your system.
+* Change parameters in start.sh, or set the environment variables ($QUEUE_PROXY_ADDRS, $GROUP_ID, $AUTHORIZATION_KEY, $TOPIC, $HTTP_HOST, $PRODUCER_HOST, $PRODUCER_HOST_HEADER, $PRODUCER_VULCAN_AUTH, $PRODUCER_TYPE) for your system.
```
Comment (tamas-molnar) [inline README.md]: Can this be a bulleted list of params please?

File: healthchecks.go [hunk 2]
```diff
@@ -9,50 +9,38 @@ import (
 	"net/http"
 )
 
-func (bridge BridgeApp) ForwardHealthcheck() fthealth.Check {
+var httpClient *http.Client = &http.Client{}
+
+func (bridge BridgeApp) ConsumeHealthcheck() fthealth.Check {
```
Comment (tamas-molnar) [inline healthchecks.go]: camel case on fthealth

File: README.md (hunk 1 above)
Comment (tamas-molnar) [inline README.md]: The possible values for the $PRODUCER_TYPE should be documented here.

**Comment by catalin-ilea** [comment]
Can you please update the bridge and use the latest message-queue-gonsumer?

File: README.md (hunk 1 above)
Comment (BerniVarga) [inline README.md]: Done

File: healthchecks.go (hunk 2 above)
Comment (BerniVarga) [inline healthchecks.go]: Done

**Comment by BerniVarga** [comment]
Yes. I made the required updates. Please check. :)

**Comment by tamas-molnar** [comment]
+1

**Comment by tosan88** [comment]
:+1:

**Comment by catalin-ilea** [comment]
lgtm

**Pull request merged**